blackgoose/jenskitchen/Recipes/pasta/couscous.html

```diff
@@ -33,6 +33,10 @@ <h1>Couscous Salad with Peas and Mint</h1>
   <li>Make the dressing in a small bowl by whisking together the juice of a lemon, the salt, pepper, and enough olive oil to properly balance the acidity of the lemon juice. It will take approximately 1/4 cup of olive oil, maybe a little more. Remember that this dressing has to flavor many cups of bland and absorbant couscous, so make sure you have enough and make sure the flavor is bold (don't be afraid of the salt... it will get spread out).</li>
   <li>When the couscous is cool, toss in the peas, mint, veggies and coat with the dressing. It can be served right away or saved until the next day.</li>
 </ol>
+  
+<p><a class = "nav" href="../salmon.html">Try this with Garlic Salmon</a></p>
+<p><a class = "nav" href="../../index.html">[Back to the home page]</a></p>
```

Comment:
Remove the class = nav parts, I forgot to take those out from my code I copy-pasted into yours.